modules/pubmaticBidAdapter.js

```diff
@@ -22,1064 +23,565 @@ const AUCTION_TYPE = 1;
 const UNDEFINED = undefined;
 const DEFAULT_WIDTH = 0;
 const DEFAULT_HEIGHT = 0;
-const DEFAULT_TTL = 360;
-const PREBID_NATIVE_HELP_LINK = 'http://prebid.org/dev-docs/show-native-ads.html';
 const PUBLICATION = 'pubmatic'; // Your publication on Blue Billywig, potentially with environment (e.g. publication.bbvms.com or publication.test.bbvms.com)
 const RENDERER_URL = 'https://pubmatic.bbvms.com/r/'.concat('$RENDERER', '.js'); // URL of the renderer application
 const MSG_VIDEO_PLCMT_MISSING = 'Video.plcmt param missing';
-
+const PREBID_NATIVE_DATA_KEY_VALUES = Object.values(PREBID_NATIVE_DATA_KEYS_TO_ORTB);
+const DEFAULT_TTL = 99;
```

Comment:
Dropping this value can have large negative effects

Reply:
This TTL is just a placeholder for the ortb converter's context object, and PubMatic SSP uses different TTL values based on the media type, as mentioned [here](https://github.com/prebid/Prebid.js/blob/86b60132a47a61c86abdb84d5f3e9000b94f6b66/modules/pubmaticBidAdapter.js#L46). I don't think it's necessary to send a default TTL here, but I'll double-check and update the PR accordingly.